ui/app/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -125,13 +125,9 @@ export default class PermissionPageContainerContent extends PureComponent {
         ],
       )
     }
-    return t(
-      'connectTo',
-      [
-        this.getAccountDescriptor(selectedIdentities[0]),
-      ],
-    )
-
+    return t('connectTo', [
```

Comment:
What motivated this change 🤔

Reply:
Whoops, habit—I'd meant to only remove the extra newline but got carried away